contracts/bridge/BridgeTestAnotherVersion.sol

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity 0.5.6;
+
+contract BridgeAnotherVersion {
```

Comment:
We need to have a better name than this. ;)

Reply:
Okay, then how about `TestBridgeContractVersionDiff`? I would appreciate it if I can get your naming consideration. This is to avoid committing multiple files again due to its renaming.

Reply:
What about `BridgeVersionController`? We can later use this also in the real bridge contract.

Reply:
BridgeVersionController is a class that has a variable VERSION and a getter function of this.

The bridge contract inherits this class, and this variable is initialized in the Bridge contract constructor.

Reply:
Okay, then do you want me to update origin contracts too so that they inherit this new change? Or, just only do rename for this function for now?

Reply:
For this PR, please do not update the original contract. Just build preparation. Then we can make another PR that update the original contract. Sounds good to you?

Reply:
Revised.